### ISSUE TITLE
[📥 Inbox] LLaVA$^3$: Representing 3D Scenes like a Cubist Painter to Boost 3D Scene Understanding of VLMs

### ISSUE THREAD

**Issue by WoojunePark** [WoojunePark/splat_papers#10]
## [LLaVA$^3$: Representing 3D Scenes like a Cubist Painter to Boost 3D Scene Understanding of VLMs](https://arxiv.org/abs/2511.16454)

**Status:** `to-read` | **Added:** 2025-11-20

### Abstract

> Developing a multi-modal language model capable of understanding 3D scenes remains challenging due to the limited availability of 3D training data, in contrast to the abundance of 2D datasets used for vision-language models (VLM). As an alternative, we introduce LLaVA$^3$ (pronounced LLaVA-Cube), a novel method that improves the 3D scene understanding capabilities of VLM using only multi-view 2D images and without any fine-tuning. Inspired by Cubist painters, who represented multiple viewpoints of a 3D object within a single picture, we propose to describe the 3D scene for the VLM through omnidirectional visual representations of each object. These representations are derived from an intermediate multi-view 3D reconstruction of the scene. Extensive experiments on 3D VQA and 3D language grounding show that our approach outperforms previous 2D-based VLM solutions.

### Links

[arXiv](https://arxiv.org/abs/2511.16454) | [PDF](https://arxiv.org/pdf/2511.16454) | [HTML](https://arxiv.org/html/2511.16454)

### Figures

![Figure 1: Overview of LLaVA3. We first reconstruct the 3D scene as a NeRF from multi-view images with an associated LLaV...](https://arxiv.org/html/2511.16454/x1.png)
*Figure 1: Overview of LLaVA3. We first reconstruct the 3D scene as a NeRF from multi-view images with an associated LLaV...*

![Figure 2: Qualitative performance of our method on 3D VQA and Grounding. By decomposing into objects the view-dependent ...](https://arxiv.org/html/2511.16454/figures/exps.jpg)
*Figure 2: Qualitative performance of our method on 3D VQA and Grounding. By decomposing into objects the view-dependent ...*

![Figure 3: Precise Architecture of the Feature Fields. NN stands for ”Normalization”.](https://arxiv.org/html/2511.16454/figures/sup-archi.jpg)
*Figure 3: Precise Architecture of the Feature Fields. NN stands for ”Normalization”.*

---

*Managed by splat-papers. Close this issue once you have read the paper. Leave comments for your reading notes — they will be synced back to `## My Notes`.*
